.travis.yml

```diff
@@ -0,0 +1,35 @@
+language: go
+
+os:
+  - linux
+
+go:
+  - "1.12"
+
+git:
+  depth: 1
+
+services:
+  - 'docker'
+
+env:
+  - GO111MODULE=on
+
+install: true
+
+jobs:
+  include:
+    - stage: unit test
+      script:
+      - curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin v1.22.2
+      - make common-check-license
```

Comment:
this needs to move to the same naming as for the core, it also requires an update of the target
we should at least make the target name change so it does not need to be done later: license-check

Reply:
Renamed to `license-check`